babel.config.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+    env: {
+        test: {
+            presets: [
+                [
+                    '@babel/preset-env',
+                    {
+                        targets: {
+                            node: 'current',
+                        },
+                    },
+                ],
+            ],
+        },
+        development: {
+            presets: [
+                [
+                    '@babel/preset-env',
+                    {
+                        modules: false,
+                    },
+                ],
+            ],
+            plugins: ['add-module-exports', '@babel/plugin-external-helpers'],
```

Comment:
Why use `add-module-exports`?

"Dropped use of add-module-exports plugin on Babel packages. This had to be used earlier to prevent a breaking change with our exports. If you import a Babel package in a library you may need to use .default when using require rather than import."
https://babeljs.io/docs/en/v7-migration-api